db/seeds.rb

```diff
@@ -14,23 +14,25 @@
 
   Faker::Config.locale = 'en-GB'
 
-  FactoryBot.create_list(:organisation, 2).each do |organisation|
-    FactoryBot.create_list(:user, 5, organisation: organisation).each do |user|
-      contact_list = FactoryBot.create :contact_list
-
-      FactoryBot.create :contact_list_user, contact_list: contact_list, user: user
-
-      contacts = FactoryBot.create_list :contact, 50, contact_list: contact_list
-
-      need_categories = ['phone triage', 'groceries and cooked meals', 'physical and mental wellbeing', 'financial support',
-                        'staying social', 'prescription pickups', 'book drops and entertainment', 'dog walking', 'other']
+  if ENV['COUNCIL'] == 'camden'
```

Comment:
Might be worth adding a bit to the README.md about this ENV var

Reply:
We have got it in the list of env vars in the readme - did you mean somewhere else?

Reply:
I meant adding a line about running the seeds with that env var, but it's probably fine without it.  `COUNCIL=camden rails db:seed` as per your comment on the PR